avm/res/service-fabric/cluster/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,234 @@
+          {
+            name: 'AppPollIntervalInSeconds'
+            value: '60'
+          }
+        ]
+      }
+    ]
+    managementEndpoint: 'https://${namePrefix}${serviceShort}001.westeurope.cloudapp.azure.com:19080'
+    reliabilityLevel: 'Silver'
+    nodeTypes: [
+      {
+        applicationPorts: {
+          endPort: 30000
+          startPort: 20000
+        }
+        clientConnectionEndpointPort: 19000
+        durabilityLevel: 'Silver'
+        ephemeralPorts: {
+          endPort: 65534
+          startPort: 49152
+        }
+        httpGatewayEndpointPort: 19080
+        isPrimary: true
+        name: 'Node01'
+
+        isStateless: false
+        multipleAvailabilityZones: false
+
+        placementProperties: {}
+        reverseProxyEndpointPort: ''
+        vmInstanceCount: 5
+      }
+      {
+        applicationPorts: {
+          endPort: 30000
+          startPort: 20000
+        }
+        clientConnectionEndpointPort: 19000
+        durabilityLevel: 'Bronze'
+        ephemeralPorts: {
+          endPort: 64000
+          startPort: 49000
+          httpGatewayEndpointPort: 19007
+          isPrimary: true
+          name: 'Node02'
+          vmInstanceCount: 5
+        }
+      }
+    ]
+    notifications: [
+      {
+        isEnabled: true
+        notificationCategory: 'WaveProgress'
+        notificationLevel: 'Critical'
+        notificationTargets: [
+          {
+            notificationChannel: 'EmailUser'
+            receivers: [
+              'SomeReceiver'
+            ]
+          }
+        ]
+      }
+    ]
+    upgradeDescription: {
+      forceRestart: false
+      upgradeReplicaSetCheckTimeout: '1.00:00:00'
+      healthCheckWaitDuration: '00:00:30'
+      healthCheckStableDuration: '00:01:00'
+      healthCheckRetryTimeout: '00:45:00'
+      upgradeTimeout: '02:00:00'
+      upgradeDomainTimeout: '02:00:00'
+      healthPolicy: {
+        maxPercentUnhealthyNodes: 0
+        maxPercentUnhealthyApplications: 0
+      }
+      deltaHealthPolicy: {
+        maxPercentDeltaUnhealthyNodes: 0
+        maxPercentUpgradeDomainDeltaUnhealthyNodes: 0
+        maxPercentDeltaUnhealthyApplications: 0
+      }
+
+    }
+    vmImage: 'Linux'
+    roleAssignments: [
+      {
+        roleDefinitionIdOrName: 'Owner'
+        principalId: nestedDependencies.outputs.managedIdentityPrincipalId
+        principalType: 'ServicePrincipal'
+      }
+      {
+        roleDefinitionIdOrName: 'b24988ac-6180-42a0-ab88-20f7382dd24c'
+        principalId: nestedDependencies.outputs.managedIdentityPrincipalId
+        principalType: 'ServicePrincipal'
+      }
+      {
+        roleDefinitionIdOrName: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'acdd72a7-3385-48ef-bd42-f606fba81ae7')
+        principalId: nestedDependencies.outputs.managedIdentityPrincipalId
+        principalType: 'ServicePrincipal'
+      }
+    ]
+    applicationTypes: [
+      {
+        name: 'WordCount' // not idempotent
```

Comment:
Quick question on this one - if the value is not idempotent, shouldn't it then maybe be moved into a dedicated test that does not perform an indempotency test? I'm just curious as this deployment here runs in a loop that is executed twice - so either it doesn't fail (which makes it idempotent) and the comment is wrong, or something is off :D Your pipeline run log for this test looks good.

Reply:
Agreed that the test passed and we saw that this comment was in the original CARML repo, so we think it can just be removed.